bin/check-graylog-buffers.rb

```diff
@@ -114,7 +114,7 @@ def call_api(path, postdata = nil)
     if !postdata
       JSON.parse(resource.get)
     else
-      JSON.parse(resource.post(postdata.to_json, content_type: :json, accept: :json))
+      JSON.parse(resource.post(postdata.to_json, content_type: :json, accept: :json, x_requested_by: SecureRandom.base64(32)))
```

Comment:
I think it would be nice to include something to indicate that it's coming from sensu. Also does this specifically need to be `base64` encoded or can we use something like a UUID? For example `x_requested_by: sensu_client: #{SecureRandom.uuid}`. Alternatively we could sue the sensu_client name I'm game for whatever as it looks like graylog does not force you to do anything beyond sending something in the header.

Reply:
I'd also prefer if we break up that line with one param per line as it now is a bit long and harder to read. Something like this:
```ruby
JSON.parse(resource.post(
  postdata.to_json,
  content_type: :json,
  accept: :json,
  x_requested_by: SecureRandom.base64(32)
))
```
Disclaimer rubocop _might_ complain about the spacing but we can deal with that later.

Reply:
Anything can be used as the value, as at the moment graylog just checks for the presence of the value then throws it on the floor. I just used a random number as I didn't have any better ideas at the time.
Sensu client name would make sense, I'll tweak it to use that.